cesium_app/handlers/model.py

```diff
@@ -61,7 +61,7 @@ def _build_model_compute_statistics(fset_path, model_type, model_params,
         raise ValueError("Cannot build model for unlabeled feature set.")
     model = MODELS_TYPE_DICT[model_type](**model_params)
     if params_to_optimize:
-        model = GridSearchCV(model, params_to_optimize)
+        model = GridSearchCV(model, params_to_optimize, n_jobs=-1)
```

Comment:
We'll want to surface n_jobs to the front-end where the parameters for training are selected. We wont always want to use all the cores since the memory footprints can be extremely high just for one core.

Reply:
Agreed that the default should be -1 but the front end might want to report the number of cores available and the let the user decide what they want to set for n_jobs.